app/jinja_filters.py

```diff
@@ -15,6 +15,8 @@
 from app.questionnaire.rules import convert_to_datetime
 blueprint = flask.Blueprint('filters', __name__)
 
+common_date_format = '%-d %B %Y'
```

Comment:
By convention, constants such as this would be defined in upper case. It's probably worth just calling it `DATE_FORMAT`